packages/medusa/src/api/routes/store/carts/update-cart.ts

```diff
@@ -173,4 +190,8 @@ export class StorePostCartsCartReq {
 
   @IsOptional()
   context?: object
+
+  @IsString()
+  @IsOptional()
+  sales_channel_id?: string
```

Comment:
Though/Idea(non-blockin): is it possible to decorate these based on feature flag?

Reply:
The only thing, is that the auto complete will still provide it even if it is decorated ... I can still find a solution if we really want to :)

Reply:
I was just thinking about mixin 😂 but let's keep that for another time